test/integration/repositories/pillars.sls

```diff
@@ -16,6 +16,7 @@ openvpn:
       comp_lzo: "yes"
       ifconfig: 169.254.0.1 169.254.0.2
       log_append: /var/log/openvpn/myserver1.log
+      status: /var/log/openvpn/myserver1-status.log
```

Comment:
I don't see why you added those `status` values to tests pillars, are they used somewhere?

Reply:
The tests would not succeed unless those values are set. So yes, they are used by the formula.